artifactory/services/utils/replication.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+type ReplicationParams struct {
```

Comment:
Lets separate the parameters and the request body.

Create `ReplicationBody` struct.
The parameters should not contain the json recipe (`json:"username"`).
Please also create a translator between the parameters and the body.